const.go

```diff
@@ -122,16 +158,15 @@ var ConfToEnv = map[string]string{
 	// [SecretStore.Authentication] -- not supported
 	// [SecretStoreExclusive] -- application service only; not supported
 
+	// TODD: replace [Binding] with [Trigger] hierarchy
```

Comment:
Does this comment come from one of the `configuration.toml` files? Has `[Trigger]` not yet been added?

Reply:
This was a comment from me... ASC no longer uses Binding, but uses Trigger instead, so this was a reminder that once we have looked at that - and updated the ASC rich configure hook map, then we need to update this as well.

Reply:
I'll update that comment to make it clearer what it's about.